src/libstd/benches/hash/set_ops.rs

```diff
@@ -0,0 +1,52 @@
+use test::{black_box, Bencher};
+use std::collections::HashSet;
+
+#[bench]
+fn set_difference(b: &mut Bencher) {
+    let small: HashSet<_> = (0..10).collect();
+    let large: HashSet<_> = (0..100).collect();
+
+    b.iter(|| {
+        black_box(small.difference(&large).count());
```

Comment:
Can be

```
small.difference(&large).count()
```

because `b.iter()` does `black_box` on closure result.

Reply:
Did not remember reading that, fixed elsewhere too.